README.md

```diff
@@ -188,6 +175,21 @@ If you want to create your custom destination it is recommended to inherit it fr
 
 If you choose to inherit from `OktaLoggerDestinationBase`, than you need to implement `log()` method in order to make it work. You can use `stringValue()` method to receive default log message string representation. Please, refer to [OktaLoggerConsoleLogger](OktaLogger/OktaLoggerConsoleLogger.swift) as an example of logger destination implementation.
 
+## OktaAnalytics
+
+![Analytics Logger Design](https://user-images.githubusercontent.com/491437/192922456-1d78ad47-8c03-4e97-8260-d08375ee54a8.svg)
+
+`OktaAnalytics` is a pod that's useful to track `Analytics`, right now we are supporting `AppCenter` and dependent on `OktaLogger` framework.
+  OktaAnalytics has functions to add, remove, purge providers and trackevent functions.
+ Steps to use `OktaAnalytics` 
+ -  Client have to implement `AnalyticsProviderProtocol` and provide information about provider (or) use create `AppCenterAnalyticsProvider`     instance by providing `AppCenterAnalytics.Analytics.Type` with `init`
```

Comment:
Extra spaces after `AppCenterAnalyticsProvider`

Reply:
Will clean up, I used soem markdome tool and not sure how these added